roles/test-beat/tasks/darwin/main.yml

```diff
@@ -25,14 +25,20 @@
     dest: '{{ beat_cfg }}'
     mode: 0600
 
+# TODO (andrewkroh on 2018-01-29): Remove -path.logs after resolving
+# https://github.com/elastic/beats/issues/6214.
 - name: 'Start {{ beat_service_name }} service'
-  command: './{{ beat_name }}'
+  command: './{{ beat_name }} -path.logs={{ beat_logs_path }}'
```

Comment:
@andrewkroh You can remove the workaround for elastic/beats#6214

Reply:
Thanks for the quick fix. The work-around has been removed.

Reply:
@andrewkroh FYI, it will work with BC5, when it happens.